src/OnlineSales/Program.cs

```diff
@@ -434,4 +450,58 @@ private static void ConfigureCORS(WebApplicationBuilder builder)
             });
         });
     }
+
+    private static void ConfigureIdentity(WebApplicationBuilder builder)
+    {
+        builder.Services.AddIdentity<User, IdentityRole>()
+            .AddEntityFrameworkStores<PgDbContext>();
+        builder.Services.ConfigureApplicationCookie(options =>
+        {
+            // Cookie settings
+            options.Cookie.HttpOnly = true;
+            options.ExpireTimeSpan = TimeSpan.FromHours(12);
+
+            options.LoginPath = "/api/Identity/ExternalLogin";
+            options.AccessDeniedPath = "/AccessDenied";
```

Comment:
@Exmirai 
1) The route does not match our naming convention
1) Where that route is supposed to be defined?